### DISCUSSION TITLE
Projects [Beta] Issue converting drafts to issues -- organization repos not visible in dropdown list

### DISCUSSION THREAD

**Discussion by mfalconi-perle** [community/community#15989] [Other Feature Feedback, Questions, & Ideas]
Hi github,

So this issue is regarding the new GitHub Projects [Beta] System. My organization has begun to move towards using this new tool, however we have encountered an issue and I cannot find a definitive solution or answer explaining the cause of the problem. I am left to wonder if this is possibly a bug with Projects [Beta] or some configuration set up we have done incorrectly.

**Issue:** after creating new Projects at the organizational level, when trying to either add a issue from a repo owned by the organization to the newly created organization project, or when trying to convert a draft issue into an issue in a certain repo, the dropdown list of selectable repositories is limited to a small subset of all the repos owned by the organization, as such we cannot add issue from certain repos into new projects & we cannot convert draft issues into issue for certain repos.
![image](https://user-images.githubusercontent.com/96071862/166485985-8982af08-f3e1-44ab-8981-7c50e68140e8.png)
As you can see here only 8 of the organizations repos can be accessed from within the Projects tables for adding issues, this also exists in the card view.

I was able to find a small work around, but still it seems like this is some sort of bug. 
**Workaround:**  If a organizational project has been generated, I am able to open an issue in a repo, and then select to add this issue to the project, which then makes the repo selectable within the specific Project. 
![image](https://user-images.githubusercontent.com/96071862/166486418-3f458536-3626-46a8-844e-b2d17de86b5a.png)
As you can see here once I have added the Project to the issue i can now view it within the Project and convert draft issues into issues for the repo.

Why is this behaviour seemingly unidirectional for certain repos in our organization?

**Comment by connorhsm** [comment]
Also experiencing this exact behaviour and confirming the workaround.

I would like to clarify the workaround, that you must add the issue to the project from the issue page.

**Comment by Bonkles** [comment]
I had this same problem- the fix is that the linkage to a Beta project board appears to be two-way. You can create a board for an organization, _but I had to make the project 'opt-in' to being part of the project board_. 

Go to your repo in your org, click 'Projects' -> 'Projects (Beta)' and use the dropdown selector to find your new beta project and add it. 


it should now look something like: 
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/1887955/176231014-4753dd9c-6ada-4afb-b8cc-39c6c1fd91ff.png">


Then grab some coffee, I had to wait 5-10 minutes for the project to 'see' the repo correctly and display it as an option when converting to an issue. 

Hope that helps!

**Reply by oliverangelil** [reply]
This solution (of first going to the repo page, and linking the project from there) didn't work for me, even after waiting for 1 hour the repo wasn't available when trying to link the draft issue to a repo.

**Comment by phillippbertram** [comment]
In my case, the problem was that I had not activated the "Issues" feature in my repository. 🤦

**Reply by zavierrodriguez** [reply]
This fixed my issue as well!  Thank you!

**Reply by tbrammar** [reply]
This also fixed the issue at our end too. It should be made clear somewhere (perhaps in the dropdown?) as I suspect many people will encounter this.

**Reply by mohamedtahaguelzim** [reply]
Thank you, fixed it ! You can activate issues in your repo's settings, under General > Features > Issues.
